tests/system/Validation/ValidationTest.php

```diff
@@ -628,6 +628,33 @@ public function testTagReplacement()
         $this->assertSame($expected, $errors['Username']);
     }
 
+    public function testRulesForObjectField()
+    {
+        $config          = new App();
+        $config->baseURL = 'http://example.com/';
+
+        $this->validation->setRules([
+            'configuration' => 'required|check_object_rule',
+        ]);
+
+        $data = (object) ['configuration' => (object) ['first' => 1, 'second' => 2]];
+        $this->validation->run((array) $data);
+        $this->assertSame([], $this->validation->getErrors());
+
+        $this->validation->reset();
+
+        $this->validation->setRules([
+            'configuration' => 'required|check_object_rule',
+        ]);
+
+        $data = (object) ['configuration' => (object) ['first1' => 1, 'second' => 2]];
+        $this->validation->run((array) $data);
```

Comment:
Same comment as above

Reply:
![image](https://user-images.githubusercontent.com/7590335/135243681-b9925083-d4ee-4959-b3af-7a036a636e3f.png)

Reply:
I convert to array because the first param must be an array or null

Reply:
I try to simulate an api rest petition.

![image](https://user-images.githubusercontent.com/7590335/135244348-f19670d0-8b53-4e05-960d-cd873a03c433.png)